doc/examples/viz_horizon.py

```diff
@@ -0,0 +1,444 @@
+# Customizing Visualization
+# ==========================
+#
+# Horizon provides many customization options:
+
+horizon(
+    tractograms=tractograms,
+    images=[(fa_data, affine, "fa.nii.gz")],
+    cluster=True,
+    cluster_thr=10.0,  # Smaller threshold = more clusters
+    random_colors="tracts",  # Random colors for each tractogram
+    bg_color=(1, 1, 1),  # White background
+    world_coords=True,
+    interactive=interactive,
+    out_png="horizon_customized.png",
+)
+
+if not interactive:
+    logger.info("Saved: horizon_customized.png")
+
+###############################################################################
+# Programmatic Control: Using the Horizon Class
+# ==============================================
+#
+# For advanced use cases, you can use the Horizon class directly instead of
+# the convenience function. This gives you more control:
+
+# Create Horizon instance
+hz = Horizon(
+    tractograms=tractograms,
+    images=[(fa_data, affine, "fa.nii.gz")],
+    cluster=True,
+    cluster_thr=15.0,
+    world_coords=True,
+    interactive=interactive,
+    out_png="horizon_class.png",
+)
+
+# Build the visualization
+hz.build_scene()
+
+# You can access the ShowManager for further customization
+# show_m = hz.build_show(show_m=None)
+
+if not interactive:
+    logger.info("Saved: horizon_class.png")
+
+
+###############################################################################
+# Summary
+# =======
+#
+# This tutorial covered:
+#
+# 1. Basic image visualization with Horizon
+# 2. Displaying multiple images
+# 3. Visualizing tractography with clustering
+# 4. Combining tractograms and images
+# 5. Filtering by length and cluster size
+# 6. Visualizing peak directions (see code comments)
+# 7. Interactive exploration with keyboard/mouse controls
+# 8. Customization options (colors, thresholds, backgrounds)
+# 9. Using the Horizon class directly for advanced control
+#
+# Key Parameters Summary
+# ----------------------
+#
+# - `tractograms`: List of StatefulTractogram objects
+# - `images`: List of tuples (data, affine, filename)
+# - `pams`: List of PeakAndMetrics objects
+# - `surfaces`: List of tuples (vertices, faces, filename)
+# - `cluster`: Enable QuickBundlesX clustering (bool)
+# - `cluster_thr`: Distance threshold for clustering in mm (float)
+# - `length_gt/length_lt`: Filter streamlines by length (float)
+# - `clusters_gt/clusters_lt`: Filter by cluster size (int)
+# - `world_coords`: Use world coordinates vs voxel coordinates (bool)
+# - `interactive`: Enable interactive window (bool)
+# - `out_png`: Output filename for snapshot (str)
+# - `bg_color`: Background color as RGB tuple (tuple)
+# - `roi_images`: Display images as ROI contours (bool)
+# - `random_colors`: Use random colors for tractograms/ROIs (str or bool)
+#
+# For more details, see the Horizon documentation and the command-line
+# interface tutorial at :ref:`viz_flow`.
+#
+# References
+# ----------
+#
+# .. footbibliography::
```

Comment:
I think you should mention the Horizon paper? Or remove the section if you do not want to write it.

Reply:
@maharshi-gor done!